mix.exs

```diff
@@ -5,7 +5,7 @@ defmodule WeatherReport.Mixfile do
     [
       app: :weather_report,
       version: "0.3.0",
-      elixir: "~> 1.13",
+      elixir: "~> 1.12",
```

Comment:
I downgraded to 12 because the ubuntu dist I'm using is apparently pretty out of date and 1.12 was the latest version it had available.

Reply:
fwiw, asdf is the way to go for versions.